### PR TITLE
fix android lib linkage, add icon and fanart, add source and license to addon.xml

### PR DESCRIPTION
**Pull request by AlwinEsch**
Bildschirmfoto von 2020-02-09 01-09-42

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -82,10 +82,11 @@ if(WIN32)
   # Windows does need to know about FFMPEG_LIBRARIES here
   list(APPEND DEPLIBS ${ICONV_LIBRARIES}
                       ${OPENSSL_LIBRARIES}
-                      ${LIBXML2_LIBRARIES}
-                      ${FFMPEG_LIBRARIES})
+                      ${LIBXML2_LIBRARIES})
 endif()
 
+list(APPEND DEPLIBS ${FFMPEG_LIBRARIES})
```
Comment (phunkyfish) [inline CMakeLists.txt]:
```
else(CORE_SYSTEM_NAME STREQUAL android)
    list(APPEND DEPLIBS ${FFMPEG_LIBRARIES})
endif()
```

Comment (phunkyfish) [reply CMakeLists.txt]: Or:
```
else(NOT OSX)
    list(APPEND DEPLIBS ${FFMPEG_LIBRARIES})
endif()
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -110,6 +108,8 @@ if(CORE_SYSTEM_NAME STREQUAL osx OR
                       ${SECURITY})
 
   list(APPEND DEPLIBS ${AUDIOTOOLBOX} ${COREFOUNDATION} ${COREMEDIA} ${VIDEOTOOLBOX} ${SECURITY})
+elseif(NOT CORE_SYSTEM_NAME STREQUAL osx)
```
Comment (phunkyfish) [inline CMakeLists.txt]: Would the elseif not always be true? I.e. is this the same?

```
else()
  list(APPEND DEPLIBS ${FFMPEG_LIBRARIES})
endif()
```

**Review by phunkyfish — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (phunkyfish) [reply CMakeLists.txt]: Also I think this commit and previous one can be squashed together.

Comment (AlwinEsch) [reply CMakeLists.txt]: Have updated an brought together

**Review by phunkyfish — APPROVED** [review]

**Pull request merged**